libs/utils/src/environment_variable.ts

```diff
@@ -35,6 +35,8 @@ export enum BooleanEnvironmentVariableName {
   SKIP_CAST_VOTE_RECORDS_AUTHENTICATION = 'REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION',
   // Disables exporting original snapshots with CVRs
   DISABLE_CVR_ORIGINAL_SNAPSHOTS = 'REACT_APP_VX_DISABLE_CVR_ORIGINAL_SNAPSHOTS',
+  // Disables hot reloading for VxMarkScan. Helps reduce performance problems when developing on VxMarkScan's low-resource hardware.
+  DISABLE_MARKSCAN_HOT_RELOAD = 'REACT_APP_VX_DISABLE_MARKSCAN_HOT_RELOAD',
```

Comment:
You can do it this way, but since we only need this flag in the vite config and not in the frontend at all this whole file can be reverted and just use `process.env. DISABLE_MARKSCAN_HOT_RELOAD === 'true'` in `vite.config.ts`.